v2/src/modules/block/components/MoreBlockDetails.vue

```diff
@@ -10,13 +10,13 @@
         </template>
         <template v-else>
             <div class="block-info mb-5">
-                <p class="text-button mb-1">Size</p>
+                <p class="text-button mb-1">{{ $t('block.size') }}</p>
                 <p class="text-info">
                     {{ props.blockDetails.blockSize.detail }}
                 </p>
             </div>
             <div v-if="props.uncleHashes && props.uncleHashes.length > 0" class="block-info mb-5">
-                <p class="text-button mb-1">Uncles</p>
+                <p class="text-button mb-1">{{ $t('block.uncle') }}</p>
```

Comment:
this will result in a 'Uncle' instead of 'Uncles' ( can you also fix [this pr](https://github.com/EthVM/EthVM/pull/1164/files)